coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/sql/PostgresSecondaryMetrics.scala

```diff
@@ -44,5 +46,55 @@ class PostgresSecondaryMetrics(conn: Connection) extends SqlSecondaryMetrics(con
           throw new Exception("Unexpected Postgres unique constraint violation!", uv)
       }
     }
+
+    val date = DateUtils.truncate(new Date(), Calendar.DATE)
+
+    val historyUpdated = using(conn.prepareStatement(
+      """UPDATE secondary_metrics_history
+            SET total_size = ?
+          WHERE store_id = ? AND dataset_system_id = ? AND date = ?
```

Comment:
Since you're retrieving a new date will this ever match?

Reply:
It also truncates the date

Reply:
Actually, why do we ever want to update an entry in the history?

Reply:
i thought about whether to keep the first data point in a date or the last.  The last involves more work but will give you a bit more information that may be relevant when you need it.

Reply:
Since this is a historical record, and is going to be logging a single row per upsert, why not hold onto every datapoint in a day? I'm not sure who is the intended consumer of this record so maybe there's no value in it.

Reply:
too much info isn't good.  I also heard future and history is in a loop.

Reply:
i can rename it to summary if we don't want to abuse history.